src/js/media-chrome-button.js

```diff
@@ -4,37 +4,33 @@ import { window, document } from './utils/server-safe-globals.js';
 
 const template = document.createElement('template');
 
-template.innerHTML = `
+template.innerHTML = /*html*/`
 <style>
   :host {
+    font: var(--media-font,
```

Comment:
At first, I was worried that moving into the shorthand property it'll cause issues if someone were to set a specific property or the font property on the element, but after trying it out, that seems unfounded and working as expected.